signer/tuf_on_ci_sign/delegate.py

```diff
@@ -40,14 +40,16 @@
 # sigstore is not a supported key by default
 KEY_FOR_TYPE_AND_SCHEME[("sigstore-oidc", "Fulcio")] = SigstoreKey
 
+TAG_KEYOWNER = "x-tuf-on-ci-keyowner"
+TAG_ONLINE_URI = "x-tuf-on-ci-online-uri"
 
 logger = logging.getLogger(__name__)
 
 
 def _get_offline_input(
     role: str,
     config: OfflineConfig,
-) -> OfflineConfig:
+) -> tuple[OfflineConfig, Key]:
```

Comment:
so I am wrong here: it is used (since otherwise we don't have the keyvalue) 

The original design is just bad and this makes it more visible. The key should likely just be a part of the OfflineConfig: `OfflineConfig.signers` is currently a `list[str]` and it probably should be a `list[tuple[str, Key | None]]` so each signer could have a key attached to them